radio/src/CMakeLists.txt

```diff
@@ -442,6 +442,7 @@ if(ARCH STREQUAL ARM)
     ppm_arm.cpp
     pxx_arm.cpp
     dsm2_arm.cpp
+    multi_arm.cpp
```

Comment:
Bitmaps.cmake should be in the cmake directory, would you paste the error here so that I can try to help you?